game/gbtd_gbr1.ksy

```diff
@@ -4,6 +4,7 @@ meta:
   encoding: ascii
   title: GBM file
   license: MIT
+  ks-opaque-types: true
```

Comment:
It'd be nice to insert a link to `lookup_object` source code.

Reply:
Added link to my implementation in the doc attribute.